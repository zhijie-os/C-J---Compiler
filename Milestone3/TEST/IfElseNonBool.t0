main()
{
    int j;
    if(j)
    {

    }
    else 
    {
        
    }
}