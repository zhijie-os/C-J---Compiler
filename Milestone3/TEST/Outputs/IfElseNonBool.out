Line 4, IF_ELSE has non BOOLEAN condition..
